CoffeeChat/View/MeetupStatusView.swift

```diff
@@ -0,0 +1,335 @@
+            return hyperlinkedText(for: "Facebook", to: facebook)
+        } else if let instagram = instagram {
+            return hyperlinkedText(for: "Instagram", to: instagram)
+        } else {
+            return unformattedText(for: "")
+        }
+    }
+
+    private func unformattedText(for string: String) -> NSMutableAttributedString {
+        return NSMutableAttributedString(string: string, attributes: unformattedStyle)
+    }
+
+    private func underlinedText(for string: String) -> NSMutableAttributedString {
+        return NSMutableAttributedString(string: string, attributes: underlinedStyle)
+    }
+
+    private func hyperlinkedText(for string: String, to link: String) -> NSMutableAttributedString {
+        let attributedString = NSMutableAttributedString(string: string, attributes: hyperlinkedStyle)
+        attributedString.addAttribute(.link, value: link, range: NSRange(location: 0, length: string.count))
+        return attributedString
+    }
+
+    // MARK: Date Manipulation
+
+    /**
+    Converts a date to a string formatted:
+    [Day of Week], [Month Number]/[Day Number] at [h:mm] [AM/PM] [Time Zone Abbreviation]
+    Will Format the date to the current Eastern Time Zone
+
+    # Example:
+    The Date for Sunday, September 27 2020 at 9:30 PM Eastern Time would become:
+    `"Sunday, 9/27 at 9:30 PM EDT"`
+    */
+    private func formatDate(_ date: Date) -> String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "M/d 'at' h:mm a"
+        dateFormatter.amSymbol = "AM"
+        dateFormatter.pmSymbol = "PM"
+        dateFormatter.timeZone = TimeZone(identifier: "America/New_York")!
+
+        let dayOfWeek = dateFormatter.weekdaySymbols[Calendar.current.component(.weekday, from: date) - 1]
+        let monthDayAndTime = dateFormatter.string(from: date)
+        let currentTimeZone = dateFormatter.timeZone.abbreviation(for: date) ?? ""
+
+        return "\(dayOfWeek), \(monthDayAndTime) \(currentTimeZone)"
+    }
+
+    /**
+    Returns true if meeting is tommorow or today. "Tommorow" is defined as one calendar day after today rather than 24 hours ahead.
+
+    # Example:
+    If today is October 4th 12:00 PM and the meeting is October 5th 12:00 PM this returns true
+    If today is October 4th 12:00 PM and the meeting is October 6th 12:00 PM this returns false
+    If today is October 4th 1:00 AM and the meeting is October 5th 11:00 PM this returns true
+    */
+    private func isTommorow(_ date: Date) -> Bool {
+        if
+            let today = Calendar.current.date(bySettingHour: 0, minute: 0, second: 0, of: Date()),
+            let meetingDate = Calendar.current.date(bySettingHour: 0, minute: 0, second: 0, of: date)
+        {
+            if let dayDifference = Calendar.current.dateComponents([.day], from: today, to: meetingDate).day {
+                return dayDifference <= 1
+            }
+        }
+
+        return false
+    }
+
+}
+
+extension NSAttributedString {
```

Comment:
Newlines? Also, if this extension is used anywhere else, I'd move it out of here. If not, then make it `fileprivate` I think. That way, if you try to use it elsewhere, you'll remind yourself that you should move it. On a similar note, if this method merits being in an extension, then so should `isTomorrow`. Ditto about it being file private.